lxd/db/node.go

```diff
@@ -463,6 +472,44 @@ func (c *ClusterTx) nodes(pending bool, where string, args ...interface{}) ([]No
 		return nil, err
 	}
 
+	rows.Close()
+
+	// Get node groups
+	sql = `SELECT node_id, cluster_groups.name FROM nodes_cluster_groups
+JOIN cluster_groups ON cluster_groups.id = nodes_cluster_groups.group_id`
+	nodeGroups := map[int64][]string{}
+
+	rows, err = c.tx.Query(sql)
+	if err != nil {
+		if err.Error() != "no such table: nodes_cluster_groups" {
+			return nil, err
+		}
+	} else {
+		// Don't fail on a missing table, we need to handle updates
```

Comment:
This comment is in the wrong place, it should be 2 lines up in the section that detects the missing table.